src/modules/phoneInput.js

```diff
@@ -227,6 +249,7 @@ class PhoneInput extends React.PureComponent {
                     className=""
                     countryOptions={countryOptions}
                     countrySelectComponent={CountryDropdown}
+                    fluid=""
```

Comment:
Why is `fluid` and empty string here?  Shouldn't it be a Boolean?

Reply:
That is on purpose because ReactPhoneInput throws a warning. I think it's good for now, but really what should happen is removing the spreading of props.

Reply:
Ok, fair enough.